src/App.css

```diff
@@ -1 +1,11 @@
-
+header {
+   background-color: black !important;
+}
+.header__title {
```

Comment:
Please don't use css files at all as you're using Material-UI. All the styles you can create within the `makeStyles`.